react/features/app/actions.js

```diff
@@ -109,6 +116,37 @@ export function appWillMount(app) {
 }
 
 /**
+ * Detects the platform of user agent and
+ * signals that platform detected.
+ *
+ * @returns {Function}
+ */
+export function detectPlatform() {
+    return dispatch => {
+        if (detectAndroid()) {
+            dispatch(_setPlatform('android'));
+        } else if (detectIOS()) {
+            dispatch(_setPlatform('ios'));
+        }
+    };
+}
```

Comment:
How does that relate to React Native's `Platform` class? You basically want us to use two separate abstractions called platform.